src/tokens/font.yml

```diff
@@ -1,20 +1,20 @@
 #
 # FONT TOKENS
-# Use these tokens to set font-weight and font-family.
+# Use these tokens to set font_weight and font_family.
 #
 
 props:
   font:
-    value: "Soleil, Helvetica Neue, Arial, sans-serif"
-  font-special:
+    value: "Soleil, Helvetica Neue, Arial, sans_serif"
```

Comment:
sans_seric got replaced by sans-serif.  This needs to be fixed so the fonts work.